### PR TITLE
Cover 2966 & 2468

### PR DESCRIPTION
**Pull request by markstory**
Add docs to cover what I took away from #2966 and #2468

### CODE REVIEW

**Comment by antograssiot** [comment]
looks good to me, I added french version

File: fr/controllers/request-response.rst [hunk 1]
```diff
@@ -528,6 +528,27 @@ chaine::
...
+facilement des flux de ressources en réponses::
+
+    $file = fopen('/some/file.png', 'r');
+    $this->response->body(function () use ($file) {
+        rewind($file);
+        fpassthru($file);
+        fclose($file);
+    });
+
+Les fonctions de rappel peuvent également renvoyer le body en tant que chaine
```
Comment (HavokInspiration) [inline fr/controllers/request-response.rst]: chaîne*

**Comment by HavokInspiration** [comment]
Except for the typo, it looks good to me

**Comment by antograssiot** [comment]
@HavokInspiration corrected

**Pull request merged**